### PR TITLE
Cleaned the gitignore and added the linux binary

### PR DESCRIPTION
**Pull request by glvr182**

### CODE REVIEW

**Comment by glvr182** [comment]
For #175

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,23 @@
-development.log
-commands.log
+# Please do not add personal files
+
+# Logs
+*.log
+
+# Extras
 extra/lgit.rb
+extra/lgit.rb
```
Comment (jesseduffield) [inline .gitignore]: a few things are duplicated here by the looks of it

Comment (glvr182) [reply .gitignore]: Woops, forgot about that, also do you want me to just do ignore the whole extra folder?
And the notes could be replaced by *.note perhaps

Comment (jesseduffield) [reply .gitignore]: hmm I don't think I use the extra folder anymore so we can just remove that line completely.
An yeah using *.note sounds good to me :)

**Comment by jesseduffield** [comment]
LGTM

**Pull request merged**